services/GoogleAdSense.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "Google AdSense",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.google.com/adsense/tc/2018/UnitedKingdom.html",
+      "select": "body"
+    },
+    "Program Policies": {
+      "fetch": "https://support.google.com/adsense/answer/48182",
+      "select": ".article-container",
+      "remove": ".action-button, .zippy"
+    },
+    "Parent Organization Privacy Policy": {
+      "fetch": "https://policies.google.com/privacy?hl=en-GB",
+      "filter": [
+        "removeUTMfromUrls"
+      ],
+      "select": "div[role=article]",
+      "remove": ".KMMDve, .TL2alc, img[src$='.svg']"
+    },
+    "Brand Guidelines": {
+      "fetch": "https://support.google.com/adsense/answer/6370057",
+      "select": ".article-container",
+      "remove": ".no-bullets"
+    },
+    "Quality Guidelines": {
+      "fetch": "https://support.google.com/webmasters/answer/35769",
+      "select": ".article-container"
+    },
+    "Acceptable Use Policy": {
+      "fetch": "https://support.google.com/adsense/answer/9335564?hl=en&ref_topic=1250104",
+      "select": ".article-container"
+    },
+    "Restricted Use Policy": {
```

Comment:
Shouldn't be `Restriction Policy` instead?

Reply:
I liked the parallel between "Restricted Use Policy" and "Acceptable Use Policy" the latter meaning what uses are accepted and forbidden, the former meaning what uses are allowed but restricted 
But if you find it unclear I can change that :)

Reply:
Ok I understand